Description: use setuptools also for the install target so the python*:Depends substitutions work
Author: Julien Puydt
Forwarded: no

--- a/setup.py
+++ b/setup.py
@@ -170,8 +170,7 @@
            'egg_info', 'easy_install', 'upload', 'install_egg_info',
             ))
 
-if len(needs_setuptools.intersection(sys.argv)) > 0:
-    import setuptools
+import setuptools
 
 # This dict is used for passing extra arguments that are setuptools
 # specific to setup
